### ISSUE TITLE
Braille feature: NVDA should be able to show different information on two braille displays at the same time

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#8109]
Dear all,

sorry for this long issue, but it is important for the reader to understand how it should work. Based on NVDA remote, NVDA could optionally run two copies at the same time which could serve two different braille outputs (i.e. in two different applications).

Here is a practical example of functions which have been scripted for Cobra screen reader for the second braille display:
* Copy + show current screen with empty lines.
* Copy + show current screen without empty lines.
* Copy + show current data of win.-clipboard.
* Copy + show selected data.
* Copy + show last 10 speechmessages, with the last at the top (used a lot in cells of tables and grids in SAP. The header is not saved in the buffer)
* Copy and show last 10 brailleoutputs (the same as with speech).
* Set begin of selection on display2.
* Set end of selection on display 2 and copy to clipboard (possible to select text from the 2nd display and paste it where needed
* Go to top of display2.
* Go to end of display2.
* Search text forwards and backwards on display2.
* Jump and speak the previous/next sentence on display 2 (very useful for big displays with 60 cells and more)
* Speak sentences on second display in a different language than the main NVDA instance is set on (very useful for translators).
* Save input on braille display 2 in a buffer, so that it can be restored after restarting the screen reader (very useful when chrashes occur).

Te script is linked with the braille settings of the screen reader, so that all available braille display drivers are shown in the script.

Comments are very appreciated.

**Comment by leonardder** [comment]
I'm closing this as a duplicate of #6614. Please add additional thoughts to that ticket.
